src/Host/Broker/Impl/Security/SecurityManager.cs

```diff
@@ -58,24 +58,37 @@ public class SecurityManager {
             IntPtr token;
             WindowsIdentity winIdentity = null;
             string profilePath = "";
+
+            _logger.LogInformation($"Attempting login for user: {context.Username}");
```

Comment:
Non-error logging here should probably use `LogTrace` instead of `LogInformation`. ASP.NET itself will log messages about authentication starting/failing at "info" level, and this is more detailed representation of that (and more than admins would normally need by default, probably), so it should go to the next verbosity level.